src/utils/validation.test.ts

```diff
@@ -26,6 +27,24 @@ describe("validateBoolean", () => {
   });
 });
 
+describe("validateInteger", () => {
```

Comment:
`nit` you're missing the happy case

Reply:
Raised https://linear.app/skynetlabs/issue/SKY-377/add-happy-cases-to-validation-functions.